api_tiltrun.md

```diff
@@ -133,27 +135,24 @@ type TargetState struct {
 ```
 
 ##### TargetStateWaiting
-
-**❓ OPEN QUESTIONS**
-* Is there any reason a target is _waiting_ to execute other than dependencies? If so, do we need `Reason`?
-  * In K8s, `Reason` has things like `ImagePullBackOff` or `CrashLoopBackOff`
-
 ```go
 // TargetStateWaiting is a target that has been enqueued for execution but has not yet started.
 type TargetStateWaiting struct {
-	// TriggerTime is when the earliest event occurred (e.g. file change) occurred that resulted in the target being
-	// enqueued.
-	TriggerTime metav1.MicroTime `json:"triggerTime"`
-	// Reason is a description for why the target is waiting and not yet active.
-	Reason string `json:"reason"`
+    // StartTime is the time at which the target entered the waiting state.
+    StartTime time.Time `json:"startTime"`
+    // Reason is a brief description of why the target has not yet started execution.
+    Reason string `json:"reason"`
```

Comment:
if this is a future field, probably take it out of the spec (so it's consistent with `TargetStateActive`)